File: service/TFC.Application.DTO/SplitDay/UpdateSplitDay/UpdateSplitDayRequest.cs
using TFC.Application.DTO.Entity;

namespace TFC.Application.DTO.SplitDay.UpdateSplitDay
{
    public class UpdateSplitDayRequest
    {
        public int? RoutineId { get; set; }
        public string UserEmail { get; set; } = string.Empty;
        public List<string> AddDays { get; set; } = new List<string>();
        public List<string> DeleteDays { get; set; } = new List<string>();
    }
}

File: tfc_gym_app/lib/data/models/dto/split_day/update_split_day_request.dart
class UpdateSplitDayRequest {
  final List<String> addDays;
  final List<String> deleteDays;
  final int routineId;
  String? userEmail;

  UpdateSplitDayRequest({
    required this.addDays,
    required this.deleteDays,
    required this.routineId,
    required this.userEmail,
  });
}